### ISSUE TITLE
Support split screen page and translation on tablet

### ISSUE THREAD

**Issue by ahmedre** [quran/quran_android#759]
A lot of this work is actually already done - the pieces left are:

1. in `TabletFragment`'s `getAyahTrackerItems`, when the mode is translation, return an arabic tracker and a translation tracker.

2. in `TabletFragment`'s `onCreateView`, call `init` with the proper page types - i.e. with `TabletView.QURAN_PAGE, TabletView.TRANSLATION_PAGE` (if the Quran page is on the right, or the opposite if the Quran page is on the left).

3. expose a switch to enable this somewhere in the ui - let's discuss ideas about this on the ticket - the very simplest would be a setting in the settings menu, which would work, but wouldn't be ideal.

4. the adapter needs to know that mixed mode is 604 pages, not 302.

miscellaneous points:

- when the Quran page is on the left, let the Quran page be on the left and the translation on the right, and vice versa.

- when refactoring the adapter, let's think about cleaning it up - maybe injecting different types of adapters that just override whatever one or two methods are extra or different instead of all the nested if/elses.

cc @ozbek and @hosainnet since one of you maybe interested in doing this - please comment before starting just to avoid duplication of work.

**Comment by Korogba** [comment]
I will take a stab at this. 

For clarification: 
- In this mode you will have just the translations without any Arabic text interleaved on one side of the page and the Arabic page on the opposite side?
- Also, has any decision been made on how the switch will be exposed? If not, I will go with the settings for now except I come up with something better.

**Comment by ahmedre** [comment]
sorry for the late reply - what i had in mind was, tablet mode already supports 2 fragments side by side - either 2 translation fragments or two ayah fragments. so in this case, we'd mix them - one translation fragment and one translation fragment (for the same page).

i'd imagine this would replace the "default" behavior for translation mode on tablets, with an option in settings to keep the old behavior (that we can eventually remove if anyone actually still uses it  ¯\_(ツ)_/¯).

**Comment by Korogba** [comment]
@benomaire Wa ʾantum fa-jazakumu-Allahu khayran. 

Looking at the link you posted, I think that is an entirely different functionality from splitting the screen on tablets into an ayah and a translation fragment. 

Do you think it will be better to create a separate issue specifically for that, if I have not misunderstood what you meant?

**Comment by benomaire** [comment]
You are right, it is a totally different functionality. I should have opened a new issue.
You can ignore that new idea for now, and lets focus on the split screen.
وجزاك الله خيرا
